zktop.py

```diff
@@ -68,11 +70,11 @@
 
 resized_sig = False
 
-def strToLong(str, base):
-    try:
-        return long(str, base)
-    except: # py3
-        return int(str, base)
+def strToLong(str, base = 10):
+    ''' Converts string into integer.
+        TODO: remove, is used only once in the entire code.
+    '''
+    return (long if sys.platform.major == 2 else int)(str, base)
```

Comment:
You're right, messed up without testing. Will be corrected in a even cleaner (?) form.